avocado/utils/network/interfaces.py

```diff
@@ -314,3 +314,29 @@ def remove_ipaddr(self, ipaddr, netmask):
         except Exception as ex:
             msg = 'Failed to remove ipaddr. {}'.format(ex)
             raise NWException(msg)
+
+    def move_interface_file(self, ignore_missing=True):
+        """
+        Revert interface file from backup
```

Comment:
All methods here in this module have the summary line on the line right after the method and are ending with a 'dot' for proper documentation rendering. Let's try to keep following this pattern here.